File: leetcode/2457/2457.ts
function unequalTriplets(nums: number[]): number 
{
    let count = 0;
    let ans = 0;
    for (let i = 0; i < nums.length - 2; i++)
    {
        for(let j = i + 1; j < nums.length - 1; j++)
        {
            for(let k = j + 1; k < nums.length; k++)
            {
                if(nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k])
                {
                    count++;
                }
            }
        }
    }
    return count;
};

File: leetcode/2457/2457.r
unequalTriplets <- function(nums) {
  length <- length(nums)
  ans <- 0
  for(i in 1:length) {
    for(j in (i + 1):length) {
      for(k in (j + 1):length) {
        if(nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k]) {
          ans <- count + 1
        }
      }
    }
  }
  return(count)
}
